src/Perlang.Stdlib/Stdlib/Libc.cs

```diff
@@ -0,0 +1,109 @@
+    /// can be found at https://github.com/perlang-org/perlang/blob/master/NOTICE.md.
+    /// </remarks>
+    [GlobalClass]
+    public static class Libc
+    {
+        // Internal class which contains the P/Invoke definitions, to avoid exposing them directly to our callers.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private static class Internal
+        {
+#if _WINDOWS
+            [DllImport("ucrtbase", EntryPoint = "_getpid")]
+#else // POSIX
+            [DllImport("libc")]
+#endif
+            public static extern int getpid();
+        }
+
+        /// <summary>Gets the fully qualified path of the current working directory.</summary>
+        /// <value>The directory path.</value>
+        /// <remarks>
+        /// By definition, if this process starts in the root directory of a local or network drive, the value of this
+        /// property is the drive name followed by a trailing slash (for example, `/` or `C:\\`). If this process starts in a
+        /// subdirectory, the value of this property is the drive and subdirectory path, without a trailing slash (for
+        /// example, `/usr/bin` or `C:\mySubDirectory`).
+        /// </remarks>
+        /// <exception cref="T:System.IO.IOException">An I/O error occurred.</exception>
+        /// <exception cref="T:System.IO.DirectoryNotFoundException">A component of the pathname no longer exists.</exception>
+        /// <returns>The current working directory.</returns>
+        public static string getcwd()
+        {
+            return Environment.CurrentDirectory;
+        }
+
+        /// <summary>Retrieves the value of an environment variable from the current process.</summary>
+        /// <param name="name">The name of the environment variable.</param>
+        /// <returns>The value of the environment variable specified by <paramref name="name" />, or <see
+        /// langword="null" /> if the environment variable is not found.</returns>
+        /// <remarks>
+        /// The  <see cref="getenv(string)"/> method retrieves an environment
+        /// variable from the environment block of the current process.
+        ///
+        /// Environment variable names are case-sensitive on Linux and macOS and case-insensitive on Windows.
+        ///
+        /// ### On Windows systems
+        ///
+        /// On Windows systems, the environment block of the current process includes:
+        ///
+        /// - All environment variables that are provided to it by the parent process that created it. For example, a
+        ///   .NET application launched from a console window inherits all of the console window's environment
+        ///   variables.
+        ///
+        ///       If there is no parent process, per-machine and per-user environment variables are used instead. For
+        ///       example, a new console window has all per-machine and per-user environment variables defined at the
+        ///       time it was launched.
```

Comment:
The formatting here is indeed a bit "off", but the weird thing is that this is _required_. By indenting this as "ordinary Markdown", the content doesn't continue the previous paragraph. :neutral_face: Really weird. I guess we should `markdig` a shot, to see if it behaves the same or differently.